README.md

```diff
@@ -1,19 +1,111 @@
-# Why Building Api
+# Why Build An API?
 
 ## Objectives
 
   1. Understand the value of providing an API
   2. Understand the design considerations when creating APIs
 
-## Outline
-
-  * Building APIs boils down to two different purposes.
-    * Building for external usage (Foursquare API, Github API, etc)
-      * Heavily documented
-      * Versioning!!
-      * Auth, rate control
-      * We're not covering this much in this unit
-    * Building for internal use
-      * Discuss the problem of internal apis and auth. Can't store a key on the client, how to do auth?
-      * CORs as a way to stop other domains from using your internal API
-      * Not going to cover auth in a major way in this unit
+## Lesson
+
+We've seen how to *consume* various APIs, now let's talk about
+*providing* one.
+
+The choice to build and provide an API boils down to one of two options:
+
+1. Your data is valuable to more than just your application, or your
+   application has valuable integration points for other
+applications to provide new and interesting services, so you build an *external* API and open it up to third-party developers.
+2. Your data is only valuable to your company, but you access it in
+   multiple ways (e.g. a web application, IOS application, and Android
+application all for your company) or have a distributed/service architecture, or you're even just using a front-end framework with Rails as an API back-end, so you build an
+*internal* API to provide common access to all internally-managed
+clients.
+
+### External API Considerations
+
+External APIs are extremely valuable to the growth of the web. Because
+of External APIs you can watch your Uber driver approach and plot your
+course with Google maps and get a text when the driver is close and pay
```

Comment:
Can we replace these "ands" with commas? I feel sort of out of breath by the end of this sentence :)